content/posts/easystats_threeyears.Rmd

```diff
@@ -0,0 +1,144 @@
+---
+title: "Happy birthday easystats! A retrospective"
+author: "easystats"
+date: 2022-01-27
+categories: ["R", "easystats"]
+tags: ["R", "easystats", "aniversary"]
+---
+
+
+```{r message=FALSE, warning=FALSE, include=FALSE}
+library(knitr)
+options(
+  knitr.kable.NA = "",
+  digits = 2,
+  dpi = 450
+)
+
+knitr::opts_chunk$set(message = FALSE, warning = FALSE)
+
+library(parameters)
+library(see)
+```
+
+```{r echo=FALSE, out.width = "200px"}
+include_graphics("https://raw.githubusercontent.com/easystats/easystats/master/man/figures/logo.png")
+```
+
+## Happy birthday *easystats*!
+
+Two years ago, which feels like yesterday, we celebrated the [**easystats**](https://easystats.github.io/easystats) project's [first anniversary](https://easystats.github.io/blog/posts/easystats_oneyear/). 
+
+Wow, those were simpler times! One could travel for pleasure, party with dozens of people and have a face-to-face conversation, and the thing that spread like a wildfire around the globe was everyone's obsession with The Weeknd's *Blinding Lights*! 
+
+At any rate, since the pandemic did not affect GitHub in any way, our free and open-source fanatic souls have made sure that this project keeps progressing, sometimes in leaps and bounds, while other times slow as the white cloud in the sky.
+
+**What has happened since? A lot!** 
+
+
+### "Go Forth and Multiply"
+
+**easystats** project, which started out as a single R package, now comprises a whopping 10 packages! These modular packages, each with its own unique purview, connect with each other to form a unifying, consistent, and most importantly, **easy-to-use** framework for statistical modeling, visualization, and reporting in R programming language! 🕸️
```

Comment:
aktchually it started out as two packages, bayestestR already sorta-existed when easystats & insight got created (hence its non-easystatsy name hehe), and some packages were already planned :)

![](https://media.giphy.com/media/xT5LMrK8zuc34a2UX6/giphy.gif)

Reply:
Yeah, but the story is easier to tell in this way... and it's not too much of "bending" history ;-)